code/game/machinery/barsigns.dm

```diff
@@ -292,6 +292,12 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/machinery/barsign, 32)
 	desc = "Honk."
 	neon_color = "#FF998A"
 
+/datum/barsign/le_cafe_silencieux
+	name = "Le Café Silencieux"
+	icon = "le_cafe_silencieux"
```

Comment:
`icon` ⟶ `icon_state`

Reply:
did the author even test this

Reply:
wait, so why exactly are linters failing?

Reply:
> incorrect, this is a datum, not an actual obj. /datum/barsign uses `icon` for this, poor naming.

Correct, this is a datum, not an actual obj. /datum/barsign uses `icon_state` for this, good naming

Reply:
> wait, so why exactly are linters failing?

You’re using the wrong var name, pull the latest master to your branch then update your code to use the right name

Reply:
Oh there we go, i have to solve this VSC stuff later though, i can't seem to fetch upstream from it at all